### PR TITLE
Workflow tracker list UI improvements

### PR DESCRIPTION
**Pull request by burieberry**
CS-2044

### CODE REVIEW

File: packages/web-client/app/components/card-pay/header/workflow-tracker/dropdown-list/index.ts [hunk 1]
```diff
@@ -3,15 +3,62 @@ import { inject as service } from '@ember/service';
...
+  }
+
+  get isScrolled() {
+    return Boolean(
+      this.currentPosition && this.currentPosition !== this.initialPosition
+    );
+  }
+
+  lockBodyScroll() {
+    let body = document.getElementsByTagName('body')[0];
```
Comment (Aierie) [inline packages/web-client/app/components/card-pay/header/workflow-tracker/dropdown-list/index.ts:43]: this is the same as `document.body`

File: packages/web-client/app/components/card-pay/header/workflow-tracker/dropdown-list/index.ts [hunk 2]
```diff
@@ -3,15 +3,62 @@ import { inject as service } from '@ember/service';
...
+    element.onblur = this.unlockBodyScroll;
+    element.onmouseover = this.lockBodyScroll;
+    element.onmouseout = this.unlockBodyScroll;
+  }
+
+  get isScrolled() {
+    return Boolean(
+      this.currentPosition && this.currentPosition !== this.initialPosition
+    );
+  }
```
Comment (Aierie) [inline packages/web-client/app/components/card-pay/header/workflow-tracker/dropdown-list/index.ts:40]: isScrolled can be determined via `this.element.scrollTop > 0` and set in the event listener, so we don't need to use the children's positions to determine whether the contents of an element are scrolled or not:

https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTop

**Review by Aierie — APPROVED** [review]
This works nicely!

File: packages/web-client/app/components/card-pay/header/workflow-tracker/dropdown-list/index.ts (hunk 2 above)
Comment (burieberry) [reply packages/web-client/app/components/card-pay/header/workflow-tracker/dropdown-list/index.ts:40]: thanks!

Comment (burieberry) [reply packages/web-client/app/components/card-pay/header/workflow-tracker/dropdown-list/index.ts:40]: > `isScrolled` can be determined via `this.element.scrollTop > 0` and set in the event listener, so we don't need to use the children's positions to determine whether the contents of an element are scrolled or not:
> 
> https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTop

However, this doesn't tell me which list is scrolling.

**Pull request merged**